### PR TITLE
Fixes #25170 - Update admin/pass param to initial_param.

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
@ekohl somehow when i pushed up my changes on https://github.com/theforeman/puppet-foreman/pull/674 it closed it and i cant reopen even force pushing so here is the updated one. I have updated all the params but the tests are failing on fixtures.

**Comment by chris1984** [comment]
@ekohl tests are green, found what I was missing.

File: spec/acceptance/foreman_basic_spec.rb [hunk 1]
```diff
@@ -33,10 +33,10 @@ class { '::apache::mod::passenger':
 
     # Actual test
     class { '::foreman':
-      repo                => 'nightly',
-      user_groups         => [],
-      admin_username      => 'admin',
-      admin_password      => 'changeme',
+      repo                        => 'nightly',
+      user_groups                 => [],
+      initial_admin_username      => 'admin',
```
Comment (ekohl) [inline spec/acceptance/foreman_basic_spec.rb]: A bit odd alignment here and in some other files. The other acceptance tests also have a bit more whitespace than I'd expect

**Comment by chris1984** [comment]
[test]

**Comment by chris1984** [comment]
@ekohl updated indent fixes and tests are green.

**Comment by chris1984** [comment]
@ekohl updated and tests are green.

**Pull request merged**